submissions/andysmokk/dom-js/js/script.js

```diff
@@ -0,0 +1,48 @@
+import movies from "../data/data.json" assert { type: "json" };
+
+const movieList = document.querySelector(".list-movie");
+const movieBox = document.querySelector(".movie-box");
+
+const getMarkupTemplate = (movie) => {
+  return `<img class="img-poster" src=${movie.Poster} alt="poster">
```

Comment:
Also, add a `title` to `alt`. It will be more accessible.

Reply:
Fixed.